types/2020-03-02/InvoiceLineItems.d.ts

```diff
@@ -59,7 +59,7 @@ declare module 'stripe' {
       /**
        * The price of the line item.
        */
-      price?: Stripe.Price | null;
+      price: Stripe.Price | null;
```

Comment:
Pretty sure these prices moving from optional to not is fine. It's the one thing that sticks out to me the most in this PR

Reply:
yeah it's fine in node because it's a TS definition and it doesn't impact other libs because it's a string/object. If it was a long or decimal that'd be breaking in dotnet :p